=== modified file 'svo (install the zip, not me).xml'
--- svo (install the zip, not me).xml	2014-05-13 07:24:37 +0000
+++ svo (install the zip, not me).xml	2014-05-28 07:38:20 +0000
@@ -10820,11 +10820,31 @@
             <integer>3</integer>
           </regexCodePropertyList>
         </Trigger>
+        <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
+          <name>svo already lit all</name>
+          <script>svo.valid.litallpipes()</script>
+          <triggerType>0</triggerType>
+          <conditonLineDelta>0</conditonLineDelta>
+          <mStayOpen>0</mStayOpen>
+          <mCommand/>
+          <packageName/>
+          <mFgColor>#ff0000</mFgColor>
+          <mBgColor>#ffff00</mBgColor>
+          <mSoundFile/>
+          <colorTriggerFgColor>#000000</colorTriggerFgColor>
+          <colorTriggerBgColor>#000000</colorTriggerBgColor>
+          <regexCodeList>
+            <string>You have no pipes that require lighting.</string>
+          </regexCodeList>
+          <regexCodePropertyList>
+            <integer>3</integer>
+          </regexCodePropertyList>
+        </Trigger>
         <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="yes" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
           <name>svo light pipes</name>
-          <script>svo.valid.litpipes(multimatches[2][2])</script>
+          <script>svo.valid.litallpipes()</script>
           <triggerType>0</triggerType>
-          <conditonLineDelta>0</conditonLineDelta>
+          <conditonLineDelta>1</conditonLineDelta>
           <mStayOpen>0</mStayOpen>
           <mCommand/>
           <packageName/>
@@ -10834,11 +10854,13 @@
           <colorTriggerFgColor>#000000</colorTriggerFgColor>
           <colorTriggerBgColor>#000000</colorTriggerBgColor>
           <regexCodeList>
+            <string>You use</string>
             <string>You light</string>
             <string>^You light (.+)\.$</string>
           </regexCodeList>
           <regexCodePropertyList>
             <integer>2</integer>
+            <integer>2</integer>
             <integer>1</integer>
           </regexCodePropertyList>
         </Trigger>
@@ -16095,7 +16117,7 @@
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
             <name>svo got voice</name>
             <script>svo.bals.voice = true
-svo.endbalancewatch("voice")
+svo.endbalancewatch("voice", true)
 raiseEvent("svo got balance", "voice")
 </script>
             <triggerType>0</triggerType>
@@ -16162,6 +16184,9 @@
               <string>^\w+ is not yet afflicted enough to be weakened in such a way\.$</string>
               <string>^You command your firelord to bring down a portion of the power of Pyradius against \w+\.$</string>
               <string>^\w+ labours under no afflictions of the body\.$</string>
+              <string>A chaos storm disregards your order.</string>
+              <string>A humbug disregards your orders.</string>
+              <string>A bloodleech disregards your orders.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>1</integer>
@@ -16179,6 +16204,9 @@
               <integer>1</integer>
               <integer>1</integer>
               <integer>1</integer>
+              <integer>3</integer>
+              <integer>3</integer>
+              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -17975,7 +18003,7 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>^You store (\d+) (.+), bringing the total to (\d+)\.$</string>
-              <string>^You store (\d+) (.+), bringing the total to (\d+), which is the maximum for that type of object\.$</string>
+              <string>^You store (\d+) (.+), bringing the total in the rift to (\d+), which is the maximum for that type of object\.$</string>
               <string>^You store (\d+) (.+?), bringing the total in the rift to (\d+)\.$</string>
             </regexCodeList>
             <regexCodePropertyList>
@@ -39203,9 +39231,10 @@
         </Trigger>
         <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
           <name>Unnamable</name>
-          <script>svo.valid.simpleunknownany(2)
+          <script>svo.startedfighting("occultist", matches[2])
 
-svo.startedfighting("occultist", matches[2])</script>
+-- the affliction time is randomised after the actual line
+tempTimer(2, [[svo.valid.simpleunknownany(2)]])</script>
           <triggerType>0</triggerType>
           <conditonLineDelta>0</conditonLineDelta>
           <mStayOpen>0</mStayOpen>

