=== modified file 'svo (install the zip, not me).xml'
--- svo (install the zip, not me).xml	2015-07-12 04:10:01 +0000
+++ svo (install the zip, not me).xml	2015-08-01 06:01:38 +0000
@@ -2048,11 +2048,9 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>The elixir heals and soothes you.</string>
-              <string>The tonic heals and soothes you.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -2090,11 +2088,9 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>The elixir flows down your throat without effect.</string>
-              <string>The tonic flows down your throat without effect.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -2112,7 +2108,6 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>Horribly, you feel the tick suck some of the elixir out of your throat.</string>
-              <string>Horribly, you feel the tick suck some of the tonic out of your throat.</string>
               <string>The symbol of sulphur inscribed about you fades away.</string>
               <string>The symbol of mercury inscribed about you fades away.</string>
             </regexCodeList>
@@ -2120,7 +2115,6 @@
               <integer>3</integer>
               <integer>3</integer>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -2691,7 +2685,7 @@
           </Trigger>
         </Trigger>
         <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="yes" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
-          <name>Applied salve/balm</name>
+          <name>Applied salve</name>
           <script/>
           <triggerType>0</triggerType>
           <conditonLineDelta>1</conditonLineDelta>
@@ -2704,7 +2698,7 @@
           <colorTriggerFgColor>#000000</colorTriggerFgColor>
           <colorTriggerBgColor>#000000</colorTriggerBgColor>
           <regexCodeList>
-            <string>^You take out some (?:salve|balm) and quickly rub it on your </string>
+            <string>^You take out some salve and quickly rub it on your </string>
             <string>1</string>
           </regexCodeList>
           <regexCodePropertyList>
@@ -2746,11 +2740,9 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>You messily spread the salve over your body, to no effect.</string>
-              <string>You messily spread the balm over your body, to no effect.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -2768,11 +2760,9 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>The salve slides off your oily skin.</string>
-              <string>The balm slides off your oily skin.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -3030,11 +3020,9 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>Your body grows extremely dense and heavy as the mass salve infuses your skin.</string>
-              <string>Your body grows extremely dense and heavy as the density balm infuses your skin.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -3123,7 +3111,7 @@
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
             <name>svo leg broken</name>
-            <script>setTriggerStayOpen("Applied salve/balm", 1)</script>
+            <script>setTriggerStayOpen("Applied salve", 1)</script>
             <triggerType>0</triggerType>
             <conditonLineDelta>0</conditonLineDelta>
             <mStayOpen>0</mStayOpen>
@@ -3163,7 +3151,7 @@
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
             <name>svo emptied salve</name>
-            <script>setTriggerStayOpen("Applied salve/balm", 1)</script>
+            <script>setTriggerStayOpen("Applied salve", 1)</script>
             <triggerType>0</triggerType>
             <conditonLineDelta>0</conditonLineDelta>
             <mStayOpen>0</mStayOpen>
@@ -3183,7 +3171,7 @@
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
             <name>svo hypothermia</name>
-            <script>setTriggerStayOpen("Applied salve/balm", 1)</script>
+            <script>setTriggerStayOpen("Applied salve", 1)</script>
             <triggerType>0</triggerType>
             <conditonLineDelta>0</conditonLineDelta>
             <mStayOpen>0</mStayOpen>
@@ -9745,11 +9733,9 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>The elixir heals and soothes you.</string>
-              <string>The tonic heals and soothes you.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -9907,11 +9893,9 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>The potion flows down your throat without effect.</string>
-              <string>The tonic flows down your throat without effect.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -9971,11 +9955,9 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>The salve slides off your oily skin.</string>
-              <string>The balm slides off your oily skin.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -10093,11 +10075,9 @@
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
               <string>The elixir flows down your throat without effect.</string>
-              <string>The tonic flows down your throat without effect.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -10358,12 +10338,10 @@
             <colorTriggerFgColor>#000000</colorTriggerFgColor>
             <colorTriggerBgColor>#000000</colorTriggerBgColor>
             <regexCodeList>
-              <string>You messily spread the balm over your body, to no effect.</string>
               <string>You messily spread the salve over your body, to no effect.</string>
             </regexCodeList>
             <regexCodePropertyList>
               <integer>3</integer>
-              <integer>3</integer>
             </regexCodePropertyList>
           </Trigger>
           <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
@@ -11575,7 +11553,7 @@
           <colorTriggerBgColor>#000000</colorTriggerBgColor>
           <regexCodeList>
             <string>Your torso feels stronger and healthier.</string>
-            <string>You may apply another salve or balm to yourself.</string>
+            <string>You may apply another salve to yourself.</string>
           </regexCodeList>
           <regexCodePropertyList>
             <integer>3</integer>
@@ -11597,7 +11575,7 @@
           <colorTriggerBgColor>#000000</colorTriggerBgColor>
           <regexCodeList>
             <string>Your torso is greatly healed, but still damaged.</string>
-            <string>You may apply another salve or balm to yourself.</string>
+            <string>You may apply another salve to yourself.</string>
           </regexCodeList>
           <regexCodePropertyList>
             <integer>3</integer>
@@ -11619,7 +11597,7 @@
           <colorTriggerBgColor>#000000</colorTriggerBgColor>
           <regexCodeList>
             <string>Your head feels stronger and healthier.</string>
-            <string>You may apply another salve or balm to yourself.</string>
+            <string>You may apply another salve to yourself.</string>
           </regexCodeList>
           <regexCodePropertyList>
             <integer>3</integer>
@@ -11641,7 +11619,7 @@
           <colorTriggerBgColor>#000000</colorTriggerBgColor>
           <regexCodeList>
             <string>Your head is greatly healed, but still damaged.</string>
-            <string>You may apply another salve or balm to yourself.</string>
+            <string>You may apply another salve to yourself.</string>
           </regexCodeList>
           <regexCodePropertyList>
             <integer>3</integer>
@@ -12699,7 +12677,7 @@
           <colorTriggerBgColor>#000000</colorTriggerBgColor>
           <regexCodeList>
             <string>The remedy burns through your chest, eliminating the spreading tendrils of a virulent heartseed.</string>
-            <string>You may apply another salve or balm to yourself.</string>
+            <string>You may apply another salve to yourself.</string>
           </regexCodeList>
           <regexCodePropertyList>
             <integer>3</integer>
@@ -12721,7 +12699,7 @@
           <colorTriggerBgColor>#000000</colorTriggerBgColor>
           <regexCodeList>
             <string>Though still frozen, you feel some slight remnants of heat returning to you.</string>
-            <string>You may apply another salve or balm to yourself.</string>
+            <string>You may apply another salve to yourself.</string>
           </regexCodeList>
           <regexCodePropertyList>
             <integer>3</integer>

