=== modified file 'svo (install the zip, not me).xml'
--- svo (install the zip, not me).xml	2015-05-12 19:26:24 +0000
+++ svo (install the zip, not me).xml	2015-05-22 22:11:41 +0000
@@ -13414,10 +13414,12 @@
           <regexCodeList>
             <string>^\w+ claps (?:her|his) hands and the retardation vibration is dampened\.$</string>
             <string>You clap your hands and the retardation vibration is dampened.</string>
+            <string>Blustering past, swirling autumnal leaves carry the retardation vibration away.</string>
           </regexCodeList>
           <regexCodePropertyList>
             <integer>1</integer>
             <integer>3</integer>
+            <integer>3</integer>
           </regexCodePropertyList>
         </Trigger>
         <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="yes" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">

